tool/cli.js

```diff
@@ -891,6 +893,54 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .action(
+    tryOrExit(async ({ options }) => {
+      if (options.verbose) {
+        console.log(chalk.green("Checking for precense of CONTENT_ROOT"));
+      }
+
+      if (!CONTENT_ROOT) {
+        throw new Error(
+          "CONTENT_ROOT not set. Please see the docs at tool/README.md"
+        );
+      }
+
+      if (options.verbose) {
+        console.log(chalk.green("CONTENT_ROOT set, generating inventory"));
+      }
+
+      const inventory = {};
```

Comment:
I'm having regrets about this structure. This is on me, but when I played with the output (provided from the gist), I found that using the paths as keys made working with the entries a little complicated. What do you think of making the top-level an array and moving the path to a property of the objects? Something like this:

```jsonc
[
  {
    "path": "/files/en-us/games/anatomy/index.md",
    "frontmatter": { /* … */ }
  },
  /* … */
]

Reply:
```json
[
  {
    "path": "/files/en-us/games/anatomy/index.md",
    "frontmatter": {
      "title": "Anatomy of a video game",
      "slug": "Games/Anatomy",
      "tags": ["Games", "JavaScript", "Main Loop", "requestAnimationFrame"]
    }
  },
  {
    "path": "/files/en-us/games/examples/index.md",
    "frontmatter": {
      "title": "Examples",
      "slug": "Games/Examples",
      "tags": ["Demos", "Example", "Games", "Web"]
    }
  }
]
```